src/index.tsx

```diff
@@ -148,6 +149,7 @@ class H5AudioPlayer extends Component<PlayerProps> {
     showSkipControls: false,
     showDownloadProgress: true,
     showFilledProgress: true,
+    showFilledVolume: true,
```

Comment:
Let's change the default value to `false` for backward compatibility.

The default value will be changed to `true` in the next major version 4.x

Reply:
Done @lhz516